grommet-leaflet-core/src/Marker.jsx

```diff
@@ -19,16 +19,14 @@ const createGrommetMarker = (
     // 'grommet-marker' class prevents leaflet default divIcon styles
     className: 'grommet-marker',
     html: ReactDOMServer.renderToString(
-      (
-        <Grommet theme={theme} background="transparent">
-          {iconProp}
-        </Grommet>
-      ) || <Pin status="unknown" />,
+      <ThemeContext.Provider value={theme}>
+        {iconProp || <Pin />}
+      </ThemeContext.Provider>,
     ),
   });
 
-  const status = iconProp ? iconProp.props.status : 'unknown';
-  const options = { title, alt, icon, status };
+  const kind = iconProp ? iconProp.props.kind : 'default';
```

Comment:
Does this have to be 'default'? Can we get away with being `undefined`? If so,
```suggestion
  const kind = iconProp?.props.kind;
```

Reply:
Done.